src/components/ExtendedReminder/index.tsx

```diff
@@ -127,19 +128,25 @@ const ExtendedReminder: FunctionComponent<ExtendedReminderProps> = ({
   } as const
 
   return (
-    <StyledContainer variant={variant} {...props}>
+    <StyledContainer
+      variant={variant}
+      aria-label={variant}
+      role="region"
+      {...props}
+    >
       <StyledBadgeContainer>
-        <Badge size="small" variant={badgeVariant[variant]}>
+        <Badge size="small" variant={badgeVariant[variant]} as="strong">
```

Comment:
Agree, that seems strange, @iManu is that normal?

Reply:
Agree this is at least a visual regression compared to current production version

Reply:
it's good for me: semantic is augmented, we want to bring this information as a "strong" and relevant data to pay attention on

You need to **adapt CSS** styles if needed because of defaults browsers styles (that are not semantic)